docs/docs/20-usage/50-environment.md

```diff
@@ -110,6 +111,7 @@ This is the reference list of all environment variables available to your pipeli
 | `CI_PREV_PIPELINE_PARENT`        | previous pipeline number of parent pipeline                                                  |
 | `CI_PREV_PIPELINE_EVENT`         | previous pipeline event (push, pull_request, tag, deployment)                                |
 | `CI_PREV_PIPELINE_URL`           | previous pipeline link in CI                                                                 |
```

Comment:
I'm a bit confused. `CI_PIPELINE_URL` was removed, but we keep `CI_PREV_PIPELINE_URL` while the new one `CI_PREV_PIPELINE_FORGE_URL` was added here as well. Is this intended?